### PR TITLE
Create PULL_REQUEST_TEMPLATE.md.

### PR DESCRIPTION
**Pull request by chrisdembia**
This PR creates a pull request template (described here: https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/). The motive is to remind people to update the change log.

To avoid adding another item to the root of the repository, I have moved the travis scripts to a `.github` directory.

### CODE REVIEW

**Comment by chrisdembia** [comment]
I stopped the appveyor builds because they are unrelated.

**Comment by jenhicks** [comment]
@chrisdembia I like this idea. It would be nice to make it more of a template that someone would fill out. 

Maybe:

Fixes # 

Brief Summary of Changes

Testing I've Completed

Looking For Feedback On

Changelog (checklist)
- Not relevant
- Updated

**Comment by chrisdembia** [comment]
I've updated the PR to follow @jenhicks' suggestion. Ready for review. I cancelled the builds.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Fixes issue #<issue_number>
+
+### Brief summary of changes
+
+### Testing I've completed
+
+### Looking for feedback on...
+
+### CHANGELOG.md (checklist)
```
Comment (jenhicks) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe "CHANGELOG.md (choose one)"

**Comment by jenhicks** [comment]
I made one suggestion. Otherwise looks good to me. We can also adjust as it is used in practice.

**Comment by chrisdembia** [comment]
> I made one suggestion.

Done. 

Please merge if you think this is good to go now.

**Comment by chrisdembia** [comment]
Thank you!

**Pull request merged**